pkg/utils/placements.go

```diff
@@ -0,0 +1,32 @@
+package utils
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func GetTolerationForCSIPods() corev1.Toleration {
+
+	runOnMaster := true
+	var err error
+	rom := os.Getenv(runCSIDaemonsetOnMaster)
+	if rom != "" {
+		runOnMaster, err = strconv.ParseBool(rom)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+	if runOnMaster {
+		toleration := corev1.Toleration{
+			Key:      "node-role.kubernetes.io/master",
```

Comment:
it looks like this is deprecated and replaced with control-plane https://kubernetes.io/docs/reference/labels-annotations-taints/#node-role-kubernetes-io-master-taint, please check what we have in supported OCP versions

Reply:
checked one of the fusion hci clusters the key was "node-role.kubernetes.io/master". we can go with this key